policy.php

```diff
@@ -0,0 +1,35 @@
+<?php
+	require_once("template.php");
+	require_once("config.php");
+	Template::render_header("Privacy Policy");
+
+	echo "<h1>" . _("Privcacy Policy") . "</h2>";
+	echo "<h2>" . _("Who we are") . "</h2>";
+	echo WHO_WE_ARE;
+	echo "<h2>" . _("Contact") . "</h2>";
+	echo POLICY_NAME . "<br>";
+	echo ADDRESS . "<br>";
+	echo POLICY_MAIL . "<br>";
+	echo POLICY_PHONE . "<br>";
+
+	echo '<h2>' . _("What personal data we collect and why") . '</h2>';
+	echo '<h3>' . _("Global") . "</h3>";
+	echo _("If you access our websites, the following information will be saved: IP-address, Date, Time, Browser queries, 
+	General information about your browser, operating system and all search queries on the sites. 
+	This user data will be used for anonym user statistics to recognize trends and improve our content.
```

Comment:
We don't do any tracking currently so I don't think it's necessary... But maybe good to have?

Reply:
It depends. For example my hosting provider has a built in stats tool and is recording these information.